build-sensor/include/step1.h

```diff
@@ -0,0 +1,16 @@
+
+class step1 {
+    public:
+        void setup() {
+            // Configure mode of LED port
+            pinMode(LED_BUILTIN, OUTPUT);
+        }
+
+        void loop() {
+            digitalWrite(LED_BUILTIN, HIGH);
```

Comment:
maybe a comment above this to explain what it's doing.

Reply:
Thanks, Done!